### PR TITLE
Add missing docu to GeometryInfo

### PR DESCRIPTION
**Pull request by peterrum**
Follow up to #10325

FYI @bangerth

### CODE REVIEW

File: include/deal.II/base/geometry_info.h [hunk 1]
```diff
@@ -2312,18 +2312,35 @@ struct GeometryInfo
...
+  standard_to_real_line_vertex(const unsigned int vertex,
+                               const bool         line_orientation = true);
 
+  /**
+   * Decompose the vertex index in a quad into a pair of a line index and a
+   * vertex index within this line.
+   */
   static std::array<unsigned int, 2>
-  standard_corner_to_line_vertex_index(const unsigned int corner);
+  standard_quad_vertex_to_line_vertex_index(const unsigned int vertex);
```
Comment (bangerth) [inline include/deal.II/base/geometry_info.h:2329]: Isn't this non-unique? That is, aren't there multiple lines that contain the vertex?

File: include/deal.II/base/geometry_info.h [hunk 2]
```diff
@@ -2312,18 +2312,35 @@ struct GeometryInfo
...
-  standard_corner_to_quad_vertex_index(const unsigned int corner);
+  standard_hex_vertex_to_quad_vertex_index(const unsigned int vertex);
 
+  /**
+   * Decompose the line index in a hex into a pair of a quad index and a line
+   * index within this quad.
+   */
   static std::array<unsigned int, 2>
-  standard_line_to_quad_line_index(const unsigned int line);
+  standard_hex_line_to_quad_line_index(const unsigned int line);
```
Comment (bangerth) [inline include/deal.II/base/geometry_info.h:2343]: same here

File: include/deal.II/base/geometry_info.h (hunk 1 above)
Comment (peterrum) [reply include/deal.II/base/geometry_info.h:2329]: Indeed. The function selects one possible one as it used to be done in `CellAccessor`. This is generalizable to TETs (see: https://github.com/peterrum/dealii/pull/7#issuecomment-630276686).

Comment (bangerth) [reply include/deal.II/base/geometry_info.h:2329]: Can you update the documentation to say that the function chooses one, and maybe which one (or say that it chooses "one" and that it is purposefully unspecified which one that is)?

Comment (peterrum) [reply include/deal.II/base/geometry_info.h:2332]: I have more or less copied the comment from `CellId`. Is that fine this way?

**Pull request merged**